parity/src/main/java/org/web3j/protocol/parity/Parity.java

```diff
@@ -26,7 +27,9 @@
     static Parity build(Web3jService web3jService) {
         return new JsonRpc2_0Parity(web3jService);
     }
-    
+
+    Request<?, ParityAddressesResponse> parityListAccounts(BigInteger quantity);
```

Comment:
Shouldn't the block parameter and address tags be here too?

https://github.com/paritytech/parity/wiki/JSONRPC-parity-module#parity_listaccounts